### PR TITLE
[runonce] Add support for user session

### PR DESCRIPTION
**Pull request by bellegarde-c**
We handle user session by prefixing targets with `user:`
    
Example: user:graphical-session.target

### CODE REVIEW

File: src/runonce-generator [hunk 1]
```diff
@@ -29,20 +29,35 @@
 RUNONCE_TARGETS="/usr/share/runonce/targets"
 RUNONCE_QUEUED_STAMP_DIRECTORY="/var/lib/runonce/queue"
 RUNONCE_QUEUED_STAMP="/var/lib/runonce/runonce_queued" # Only used by the systemd-generator
+RUNONCE_QUEUED_USER="droidian"
```
Comment (g7) [inline src/runonce-generator]: can we determine that automatically? Perhaps by looking at the groups of the user (like, if the user has UID >= 1000 and a shell configured, it's likely the target user).

This I think might also help us in future if we want to enable multiple users

Comment (bellegarde-c) [reply src/runonce-generator]: Yes, should be easy to add, will look at it tomorrow.

Comment (bellegarde-c) [reply src/runonce-generator]: Code updated to handle multiple users.

File: debian/runonce.install [hunk 2]
```diff
@@ -1,3 +1,5 @@
 src/runonce.sh /usr/share/runonce/
 src/runonce-generator /usr/lib/systemd/system-generators/
-systemd/runonce@.service /usr/lib/systemd/system/
+src/include.sh /usr/libexec/runonce/
```
Comment (g7) [inline debian/runonce.install]: Can we move this to /usr/share/runonce/ so it's in the same dir as runonce.sh ?

**Review by g7 — COMMENTED** [review]
Minor nitpick, but looks good to me. Thanks a lot!

File: debian/runonce.install (hunk 2 above)
Comment (bellegarde-c) [reply debian/runonce.install]: Done

**Review by g7 — APPROVED** [review]
Thanks!

**Pull request merged**